src/main/java/org/qubership/itool/cli/AbstractCommand.java

```diff
@@ -77,7 +90,11 @@ protected Future<?> runFlow(Vertx vertx, FlowMainVerticle main, GraphService gra
     @SuppressWarnings({ "unchecked", "rawtypes" })
     protected void configLoaded(Vertx vertx, FlowMainVerticle main, GraphService graphService, JsonObject config, Promise promise) {
         try {
-            FlowContextImpl flowContext = new FlowContextImpl(graphService);
+            // Create application context with the loaded config and custom modules
+            ApplicationContext context = new ApplicationContext(vertx, config, createModules(vertx));
```

Comment:
If you create context per command, it might result in duplicating a lot of classes as multiple commands might share the same services which are not easy to share across contexts.

WDYT of moving this closer to `public static void main`?